### PR TITLE
Updated Dead links in integrations.rst

### PR DESCRIPTION
**Pull request by ArmanChand**

### CODE REVIEW

File: source/about/integrations.rst [hunk 1]
```diff
@@ -72,9 +72,9 @@ Custom applications using APIs and drivers (private and public cloud)
 
 Mattermost provides complete access to `server APIs <https://api.mattermost.com/>`__, along with language-specific drivers to integrate into your own applications. You can also draw from dozens of open source applications to build your own.
 
-- Learn about `Mattermost REST APIs <https://api.mattermost.com/>`__.
-- Learn about `Mattermost drivers, webhooks, and slash commands <https://docs.mattermost.com/developer/api.html>`__.
-- Learn about `Mattermost open source apps and integrations <https://integrations.mattermost.com/>`__.
+- Learn about `Mattermost REST APIs <https://docs.mattermost.com/developer/api.html>`__.
```
Comment (cwarnermm) [inline source/about/integrations.rst:75]:
```suggestion
- Learn about `Mattermost REST APIs <https://api.mattermost.com/>`__.
```

Comment (cwarnermm) [reply source/about/integrations.rst:75]: I've provided an updated URL.

File: source/about/integrations.rst [hunk 2]
```diff
@@ -72,9 +72,9 @@ Custom applications using APIs and drivers (private and public cloud)
...
+- Learn about `Mattermost drivers, webhooks, and slash commands <https://docs.mattermost.com/developer/slash-commands.html>`__.
```
Comment (cwarnermm) [inline source/about/integrations.rst:76]:
```suggestion
- Learn about `Mattermost drivers, webhooks, and slash commands <https://developers.mattermost.com/integrate/slash-commands/>`__.
```

Comment (cwarnermm) [reply source/about/integrations.rst:76]: I've provided an updated URL.

**Review by cwarnermm — COMMENTED** [review]
Thanks for your docs contribution! I've provided updated URLs inline.

File: source/about/integrations.rst (hunk 1 above)
Comment (ArmanChand) [reply source/about/integrations.rst:75]: okay

File: source/about/integrations.rst (hunk 2 above)
Comment (ArmanChand) [reply source/about/integrations.rst:76]: thanks @cwarnermm

**Comment by ArmanChand** [comment]
@cwarnermm kindly merge this pr

File: source/about/integrations.rst (hunk 1 above)
Comment (justinegeffen) [inline source/about/integrations.rst:75]:
```suggestion
- Learn about `Mattermost REST APIs <https://api.mattermost.com/>`__.
```

File: source/about/integrations.rst (hunk 2 above)
Comment (justinegeffen) [inline source/about/integrations.rst:76]:
```suggestion
- Learn about `Mattermost drivers, webhooks, and slash commands <https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/>`__.
```

**Review by justinegeffen — COMMENTED** [review]
Two links were incorrect and have been corrected, awaiting your review. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thank you for your contribution! :)

**Pull request merged**